include/jsonbuilder/JsonBuilder.h

```diff
@@ -1479,6 +1545,42 @@ Exchanges the contents of two JsonBuilder objects.
 */
 void swap(JsonBuilder&, JsonBuilder&) throw();
 
+/*
+UUID object, network byte order (big-endian), compatible with uuid_t.
+Does NOT use the same byte order as the Windows GUID type.
+*/
+struct UuidStruct
+{
+    char unsigned Data[16]; // Compatible with uuid_t from libuuid.
```

Comment:
Note: `uuid_t` is a typedef for `char unsigned[16]`, so this is precisely compatible with `uuid_t`, but we don't need to `#include <uuid.h>` to get it.